packages/fxa-auth-server/lib/senders/emails/renderer.ts

```diff
@@ -7,8 +7,10 @@ export const context = {
   onDesktopOrTabletDevice: true,
   anotherDeviceURL:
     'http://localhost:3030/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device',
-  iosURL: '',
-  androidURL: '',
+  iosURL:
+    'https://accounts-static.cdn.mozilla.net/product-icons/apple-app-store.png',
```

Comment:
Added these URLs in here since [this PR](https://github.com/mozilla/fxa/pull/9788) got merged

Reply:
Wait - are `iosURL` and `androidURL` meant to be URLs to the badge images, or URLs that a user can click on from the image badge anchor? Because I still see the old URL's being used in `packages/fxa-auth-server/lib/senders/emails/partials/appBadges.ts`, and on main I [can see](https://github.com/Yogita98/fxa/blob/main/packages/fxa-auth-server/lib/senders/templates/partials/appBadges.html#L3:L4) that those variables are being used as anchors (not image URLs).

Reply:
Ohhh my bad. Previously, I had left them empty so do you want me to keep as it was before or include the url from [config](https://github.com/mozilla/fxa/blob/60282403d3aecb61d95fe98e92ca8c959c0990d5/packages/fxa-auth-server/config/index.ts#L284)?